test/e2e/logicModules/KPIRewarderLifecycle.t.sol

```diff
@@ -303,7 +303,7 @@ contract LM_PC_KPIRewarder_v1Lifecycle is E2ETest {
             (REWARD_DEPOSIT_AMOUNT - totalDistributed)
         );
 
-        // TODO: Fix Staking precision bug
+        // ::@todo: Fix Staking precision bug ::@0xNuggan Still TODO lol, In all seriousness maybe we want to put a final pin in this. It was "just" a precision issue of less than 0,000001 percent or something right?
```

Comment:
@0xNuggan I still need your feedback here

Reply:
Yeah I mean if we don't do stuff it stays as TODO :sweat_smile:  I'd really like to wait for auditor feeback on this, even small amounts can fuck up protocols big time. Did they already say something @marvinkruse  ?

Reply:
I would like to ideally remove all todos from the code and convert them into Issues in Linear
I think tasks should be contained in a Task Management Software and not in Code :see_no_evil:

Reply:
I mean, SC-193 is tracking this. We can remove the comment then

Reply:
Changed it accordingly